### ISSUE TITLE
Tracker clobbers focus of any iframe on the page

### ISSUE THREAD

**Issue by AndrewRayCode** [vincepare/iframeTracker-jquery#2]
super neat!

If you have a rich text (contentEditable iframe) editor that is not tracked, and you load the iframe tracker, every time you move your mouse out of the text editor, it blurs it, because your mousemove listener is applied globally

wat

**Comment by grzegorzjudas** [comment]
```javascript
                        var self = this;
			$(document).mousemove(function(e){
				if(!self.handlersList.some(function(el) {
					return $(el.target).is($(document.activeElement));
				})) return;

				if (document.activeElement && document.activeElement.tagName == 'IFRAME') {
					$.iframeTracker.focusRetriever.focus();
					$.iframeTracker.focusRetrieved = true;
				}
			});
```

**Comment by RobertDaleSmith** [comment]
I can confirm grzegorzjudas's solution works.

**Comment by vincepare** [comment]
Although grzegorzjudas's patch seems to work, it is wrong because it breaks click detection if you click an iframe after clicking another iframe, because the blur event is only fired for the first click. And in fact, this comes down to remove the mousemove listener.

I fixed this issue on 2.1.0 by replacing the mousemove listener by a "boundary" event at managing the focus retriever.
